tools/packaging/common/envoy_bootstrap.json

```
@@ -602,14 +602,14 @@
               }
             }
           },
-          "call_credentials": {
+          "call_credentials": [
             "sts_service": {
               "token_exchange_service_uri": "http://localhost:{{ .sts_port }}/token",
               "subject_token_path": "/var/run/secrets/tokens/istio-token",
               "subject_token_type": "urn:ietf:params:oauth:token-type:jwt",
               "scope": "https://www.googleapis.com/auth/cloud-platform",
             }
-          }
+          ]
```

```suggestion
          "call_credentials": [{
            "sts_service": {
              "token_exchange_service_uri": "http://localhost:{{ .sts_port }}/token",
              "subject_token_path": "/var/run/secrets/tokens/istio-token",
              "subject_token_type": "urn:ietf:params:oauth:token-type:jwt",
              "scope": "https://www.googleapis.com/auth/cloud-platform",
            }
          }]
````

Current form is not valid json

---

Fixed